CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+
+## [5.0.0]
```

Comment:
so since we are actually doing this to prep for a release we should include the release date as well. Also can we put a newline between `### Breaking Changes` and `### Added` headers to be consistent with the existing stuff?

Reply:
fixed